cloudformation/stackcopy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash 
+echo Enter Date in from yyyy-dd-mm
+read date 
+echo $date 
+
+
+
```

Comment:
Do you have a linter? I'm happy to help you set one up :) the computer takes care of getting rid of double spaces, too many lines, and extra whitespace (spaces after a line).

3 empty lines feels like too many spaces. Maybe 2 would be best here.

Reply:
I don't have this installed yet, but I can add this on my computer today.